torchtitan/train.py

```diff
@@ -82,6 +87,8 @@ def main(job_config: JobConfig):
 
     # build dataloader
     tokenizer = train_spec.tokenizer_cls(job_config.model.tokenizer_path)
+    dp_rank = dp_degree * job_config.experimental.ft_replica_id + dp_rank
+    dp_degree = dp_degree * job_config.experimental.ft_group_size
```

Comment:
does the `ft_group_size` affect the tps? I don't think it currently does, but should it / how should it play into the overall metrics?

Reply:
`dp_degree` will be used by dataloader to decide how to get the right dataset partition.